src/views/landing.vue

```diff
@@ -75,11 +75,13 @@ export default class Landing extends Vue {
     const { id } = this
     // if an id was specified then get and load the subject NR
     if (id) {
-      this.$root.$emit('showSpinner', true)
```

Comment:
This is the old code which showed the spinner while loading the NR, but there is a delay before this happens (ie, while Vue loads all the pages) so the user saw the initial page for a while before this code executed.
Now the spinner is shown as soon as the app loads (ie, before Vue loads all the pages).